drivers/gpu/drm/panel/panel-waveshare-dsi-v2.c

```diff
@@ -1707,6 +1710,9 @@ static int ws_panel_init_dcs_cmd(struct ws_panel *ts)
 	struct drm_panel *panel = &ts->panel;
 	int i, err = 0;
 
+	/* Allow panel controller to stabilize after power-up/reset */
+	msleep(WS_DSI_RETRY_DELAY_MS);
```

Comment:
You've already had a `msleep(60)` in `ws_panel_prepare` after setting reset high. Extend that delay if you view that to be too short.